enrichgraph/enrichgraph.go

```diff
@@ -0,0 +1,23 @@
+package enrichgraph
```

Comment:
you can move `enrichgraph` package inside of `entrich` since its the only usage for it.

Reply:
ok